src/compas_timber/consumers/geometry.py

```diff
@@ -43,7 +43,10 @@ def __init__(self, assembly):
     def result(self):
         for beam in self.assembly.beams:
             geometry = Brep.from_box(beam.blank)
-            features_geo = self._apply_features(geometry, beam.features)
+            try:
```

Comment:
@chenkasirer I'm sure your implemenation is better. will need to reject this change on merge